packages/storybook-readme/src/services/transformEmojis.js

```diff
@@ -0,0 +1,19 @@
+const emoji = require('node-emoji');
```

Comment:
Need to evaluate the weight, security, maintenance of this package, and whether it's included in the runtime.

Reply:
https://bundlephobia.com/result?p=node-emoji@1.8.1

Reply:
What do you think @tuchk4 ?

Reply:
There is 27Kb (minified) SON file with string-placeholder to emoji map.  
It is not possible to make it lesser. Only if restrict the used emoji :)

https://github.com/omnidan/node-emoji/blob/master/lib/emoji.json